### PR TITLE
Lazy load components

### PR DESCRIPTION
**Pull request by dem4ron**
- Move internal things to internal
- Lazy load components in internal.tsx
- Fix imports

### CODE REVIEW

**Comment by iHiD** [comment]
Ooooh, green 🥳

**Review by iHiD — CHANGES_REQUESTED** [review]
Currently causes bad things to happen! Video on Slack :)

File: app/javascript/components/community/video-grid/useVideoGrid.tsx [hunk 1]
```diff
@@ -55,9 +55,9 @@ export function useVideoGrid(
     usePaginatedRequestQuery<APIResponse>(
       [
         'community-video-grid-key',
-        request.query.criteria,
-        request.query.videoTrackSlug,
-        request.query.videoPage,
+        request.query.criteria || '',
+        request.query.videoTrackSlug || '',
+        request.query.videoPage || 1,
```
Comment (iHiD) [inline app/javascript/components/community/video-grid/useVideoGrid.tsx]: What I'd like to understand is what's changed in this PR to cause this to hurt us?

Comment (ErikSchierboom) [reply app/javascript/components/community/video-grid/useVideoGrid.tsx]: It might me the pagination change

Comment (ErikSchierboom) [reply app/javascript/components/community/video-grid/useVideoGrid.tsx]: I'll double check. I tried to revert a change there.

Comment (ErikSchierboom) [reply app/javascript/components/community/video-grid/useVideoGrid.tsx]: Found it: https://github.com/exercism/website/pull/5907/commits/778c9e86bc192b41b0bb1032cf7ac46912c0f02f

**Pull request merged**